Cargo.toml

```diff
@@ -12,6 +12,7 @@ members = [
   "xtra-bitmex-price-feed",
   "bdk-ext",
   "model",
+  "bitcoin-transaction-monitor",
```

Comment:
```suggestion
  "btsieve",
```

🤪

Reply:
NOOOOOOOOOOOOOOO! :joy:

Reply:
🤡

Reply:
Fine, long live btsieve! I'll change the name.

Reply:
`btc-sieve`?

Reply:
Nice one @klochowicz :D